package.json

```diff
@@ -57,14 +56,15 @@
     "@types/request": "^2.47.0",
     "@types/through2": "^2.0.33",
     "assert-rejects": "^1.0.0",
-    "gaxios": "^1.2.2",
     "codecov": "^3.0.4",
+    "gaxios": "^1.2.2",
     "gts": "^0.9.0",
     "intelli-espower-loader": "^1.0.1",
     "is-stream": "^1.1.0",
     "mocha": "^5.2.0",
     "mockery": "^2.1.0",
     "nock": "^10.0.0",
+    "node-fetch": "^2.3.0",
```

Comment:
It doesn't look like we use this, is it needed?

Reply:
Nope, it's outta here!